node_modules/oae-core/manageaccess/manageaccess.html

```diff
@@ -24,7 +24,11 @@ <h3 id="manageaccess-modal-title">__MSG__MANAGE_ACCESS__</h3>
                             <span id="manageaccess-overview-shared-title"><!-- --></span>
                         </h4>
                         <div id="manageaccess-overview-shared" class="well">
-                            <ul id="manageaccess-overview-selected" class="oae-list oae-list-compact"><!-- --></ul>
+                            <ul id="manageaccess-overview-selected-shared-container" class="oae-list oae-list-compact manageaccess-overview-selected"><!-- --></ul>
+                        </div>
+                        <div id="manageaccess-overview-invitations" class="well hide">
```

Comment:
Weird, I had a style bound to a `h3` that should have been bound to a `div.oae-tile-title`. I wonder where I got `h3` from. Maybe it was a functional merge conflict.